### ISSUE TITLE
Equivalence between implied and explicit licenses

### ISSUE THREAD

**Issue by augusto-herrmann** [ppKrauss/dataset_licenses#7]
The reports in this repository on implied licenses argue that, when some legislation puts content in some form of public domain, it is to be considered equivalent to [CC0](https://creativecommons.org/choose/zero/).

This is, however, not how Creative Commons Zero is supposed to work. While the end result may be quite similar, the process is very different. In order for some work to be licensed under CC0, the licensor must first be legally the holder of some form of copyright over said work. For cases where copyright over the work is non-existent, or otherwise the work is in the public domain by law or by definition, one must use instead the [Public Domain Mark](https://creativecommons.org/publicdomain/mark/1.0/). The party using the Public Domain Mark is not an author, licensor or holder of the work, and is labelled as just an "identifier" instead, i.e. someone has done some sort of analysis on the provenance of said work and has determined that is is indeed in the public domain.

This difference is documented in the [Creative Commons FAQ](https://wiki.creativecommons.org/wiki/CC0_FAQ#What_is_the_difference_between_CC0_and_the_Public_Domain_Mark_.28.22PDM.22.29.3F) and they also have a [chart comparing CC0 to PDM](https://wiki.creativecommons.org/wiki/CC0_PDM_comparison_chart).

In conclusion, considering Creative Commons own orientation, implied licenses where some work is determined to be in the public domain because of some legislation should be deemed equivalent to the PDM, not CC0.

**Comment by ppKrauss** [comment]
@augusto-herrmann, thanks the feedback!

I agree that the correct is "Public Domain Mark" and not "CC0 license"... In few days I will review, you not the firt to notice de problem, perhaps needs a correction in presentation, instructions and some other reports. 

---

Explaining the context and objectives.

The aim  of reports like [this one in the reports folder](https://github.com/ppKrauss/licenses/blob/master/reports/implied-lex-BR-v1.md), is to check, assert and open to endorse, inferences about **"functional equivalence"** of other kind of  "grant copy permissions" (other things that are not explicit [licenses](data/licenses.csv), as the Berne automatic grants or an local country law refering a class of documents).

Today, for "ordinary citizens" (looking for content at Web to copy/paste fragments, copy and distribute, etc.), and for functional aspects in content repositories, as [JATS format](http://jats.nlm.nih.gov/publishing/tag-library/1.1/element/license.html) (used in millions of documents in the PubMed Central, SciELO, etc.) or other formats at [N-Lex](http://eur-lex.europa.eu/n-lex/), is necessary to _translate the grants by a standard license_. 
The key to all "ordinary people" is [the _standard license_ analogy](https://github.com/ppKrauss/licenses/blob/master/data/families.csv).

So, this is the semantic context for "functional equivalence"... **We need to point a license**, even when it not is explicited by the autor, but **a kind of grant exist**.
